doc/contributor/howto-guide-declare-a-library-ga.md

```diff
@@ -0,0 +1,93 @@
+# How-to Guide: Declare a library as GA
+
+This document describes the steps required to promote a `google-cloud-cpp`
+library to GA. The document is intended for contributors to the
+`google-cloud-cpp` libraries, it assumes you are familiar with the build systems
+used in these libraries, and that you are familiar with existing libraries.
+
+This document applies to both hand-crafted and generated libraries, but mostly
+it will be using generated libraries as examples.
+
+## Overview
+
+Largely declaring a library "GA" is a matter of updating the documentation and
```

Comment:
s/Largely declaring ... is a matter/Declaring ... is largely a matter/

Reply:
Done.